### PR TITLE
dash/first-dash-docs

### PR DESCRIPTION
**Pull request by karolkolodziej**
Attempt to fix the first dashboard docs.
Looks like the layout is broken: https://www.highcharts.com/docs/dashboards/your-first-dashboard

### CODE REVIEW

File: docs/dashboards/your-first-dashboard.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ If you want to use Highcharts charts, you should also import the charts styles,
     @import url("https://code.highcharts.com/dashboards/css/dashboards.css");
     ```
 
-4. With that, we are ready to start working on an actual dashboard config. Inside the script tag, create a new instance of the dashboard. <br>
-To do so, you have to pass two arguments:
+4. With that, we are ready to start working on an actual dashboard config. Inside the script tag, create a new instance of the dashboard.  
+To do so, you have to pass two arguments:
```
Comment (sebastianbochan) [inline docs/dashboards/your-first-dashboard.md:31]: Remove extra spaces.

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

**Review by Vazonik — APPROVED** [review]
Looks ok, except the extra spaces.

**Review by Vazonik — APPROVED** [review]
Looks ok, except the extra spaces.

**Comment by karolkolodziej** [comment]
With the double space, I'm trying to fix the issue visible here:
image

I don't see this in VSCode, it is only visible in the released version thus I'm experimenting here slightly 😅

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**